packages/flutter/lib/src/semantics/semantics.dart

```diff
@@ -211,6 +213,38 @@ sealed class _DebugSemanticsRoleChecks {
     node.visitChildren(validateRadioGroupChildren);
     return error;
   }
+
+  static FlutterError? _semanticsMenu(SemanticsNode node) {
+    if (node.childrenCount < 1) {
+      return FlutterError('a menu cannot be empty');
+    }
+
+    return null;
+  }
+
+  static FlutterError? _semanticsMenuBar(SemanticsNode node) {
+    if (node.childrenCount < 1) {
+      return FlutterError('a menu bar cannot be empty');
+    }
```

Comment:
also check if all children are menu?

Reply:
Based on the [description](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Roles/menubar_role#:~:text=A%20menubar%20contains,are%20not%20focusable.) on the mdn website, a menubar's children can be `menuItem`, `menuItemCheckbox`, `menuItemRadio`, `group`, and `separator`. It looks pretty flexible, so I didn't add more checks for `menu` and `menuBar`, is this okay?